README.md

```diff
@@ -17,9 +17,9 @@ Installation
 
 We recommend the [Anaconda](https://www.anaconda.com/download/) Python
 distribution. We require that you use Python 3.
-Besides `numpy`, `scipy`, `matplotlib`, and `pandas` (which are
-included in the standard Anaconda installation), you will need to install the
-most recent version of MNE using the `pip` tool:
+Besides `numpy`, `scipy` and `matplotlib` (which are included in the standard
+Anaconda installation), you will need to install the most recent version of MNE
+using the `pip` tool:
```

Comment:
actually just realized this is not true. If you use the stable version of `mne-bids` you will still need `pandas`. Maybe you can add instructions for the development version after I do my pass